frontend/src/components/learner/browser/CourseCard.tsx

```diff
@@ -63,13 +77,13 @@ const CourseCard = (): React.ReactElement => {
               "-webkit-box-orient": "vertical",
             }}
           >
-            Description description description description description
-            description description
+            {hidden ? "Description..." : course.description}
```

Comment:
How about "No description provided"? That way, it doesn't seem like an accident.